salt/journal-cms/init.sls

```diff
@@ -274,18 +274,30 @@ aws-credentials-www-data:
 
 # populates data into the labs and subjects until they will be created through the user interface
 # TODO: this should fail, but it doesn't because drush fails silently with 0 return code
-# TODO: this needs some legacy database to be restored on this machine to be able to work, ubr should do that
+
+{% if pillar.elife.env in ['end2end', 'prod'] %}
 migrate-content:
+    # this is working through legacy databases manually restored
+    # on end2end and prod machines
+    cmd.run:
+        - name: |
+            echo "Not ready for now"
+            #../vendor/bin/drush mi --all --execute-dependencies
```

Comment:
This should be ready. There is no harm in running it. It will just report no migrations until we manually or automatically import the `legacy_cms` database and position the `config/legacy_cms_files`.